package.json

```diff
@@ -1,15 +1,29 @@
 {
   "dependencies": {
     "@code-hike/mini-terminal": "^0.1.1",
-    "next": "^9.3.6",
+    "@code-surfer/standalone": "^3.1.1",
+    "@emotion/core": "10.0.7",
+    "@mdx-js/mdx": "^1.2.2",
+    "@mdx-js/react": "^1.0.6",
+    "@testing-library/dom": "^7.5.7",
+    "@testing-library/user-event": "^10.3.1",
+    "next": "^9.4.0",
+    "raw-loader": "^4.0.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-range": "^1.6.1",
+    "react-youtube": "^7.11.2",
+    "theme-ui": "0.2.31",
     "use-spring": "^0.2.3"
   },
   "scripts": {
     "dev": "next",
     "build": "next build",
     "start": "next start"
+  },
+  "devDependencies": {
+    "@types/node": "^14.0.5",
+    "@types/react": "^16.9.35",
+    "typescript": "^3.9.3"
```

Comment:
pin versions for less surprises.

Reply:
Also Prettier is missing from the dev deps :D

Reply:
will do, thanks